dist/tools/emul8/README.md

```diff
@@ -0,0 +1,24 @@
+# Emulation using Emul8
+
+## Introduction
+[Emul8](http://emul8.org) is an open source embedded systems emulation framework. It offers support for a broad range of processors, including ARM, ARM Cortex, x86 and more. In addition it is an extensible framework that can also emulate peripherals.
+
+## Installation
+
+### From package
+At the time of writing (November 2017), no packages are available.
+
+### From source
+Follow the installation instructions on their [GitHub](https://github.com/emul8/emul8#installation) page.
+
+After compilation is successful, ensure that `emul8` is available on your `PATH`. One way to do so, is symlinking `path/to/emul8/repository/run.sh` to `/usr/local/lib/emul8`.
```

Comment:
maybe add the command: `ln -s path/to/emul8/repository/run.sh /usr/local/lib/emul8`.

Reply:
Is it possible to 'automatize' the installation of emul8 in the tools ? I'm thinking to something similar to edbg or bossa tools for flashing the boards.

Reply:
Since emul8 requires mono from another source than the usual Ubuntu/Debian sources this could become quiet complicated and/or dangerous.